### DISCUSSION TITLE
How about this, the network uses an anvil

### DISCUSSION THREAD

**Discussion by mhmmadexsal** [Cyfrin/foundry-full-course-cu#2237] [General]
![Screenshot 2024-08-14 110933](https://github.com/user-attachments/assets/20d1aa55-a5f8-4da3-9901-802614ee3bd4)

button does not work

**Comment by EngrPips** [comment]
Hello @mhmmadexsal, The error says `Invalid RPC URL` so I am prompted to ask do you have an anvil chain actively running.

**Comment by mhmmadexsal** [comment]
@EngrPips whether the anvil chain should stop ?

**Reply by EngrPips** [reply]
The `anvil` chain is supposed to keep running while you process your transactions.

**Reply by mhmmadexsal** [reply]
@EngrPips Transaction is running fine but my chrome is like that

**Reply by EngrPips** [reply]
Check that your script is using the correct `rpc-url`

**Reply by mhmmadexsal** [reply]
thank you very much, the error has disappeared

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out keep crushing it.
